.linelint.yml

```diff
@@ -1,3 +1,4 @@
+---
 # 'true' will fix files
 autofix: true
```

Comment:
this seems to be particularly dangerous and bound to create noise. should be left to the developer to run the utility with autofixing.

Reply:
This was not done by me but It is enforcing single new line at end of files. Want me to remove it ?

Reply:
Can you check if this actually does anything? if the linter runs in the Github Workflow, it is modifying the files that have been checked out. It would need to create a new commit for having these line fixes included? is it really doing that?

.github/workflows/workflow-repo-standards-validation.yml
https://github.com/fernandrone/linelint/

Reply:
It is currently being called by the reusable workflow workflow-repo-standards-validation.yml. Looking at a triggered workflow on finesse-frontend by example, I see : 
![image](https://github.com/ai-cfia/github-workflows/assets/73133692/b689670b-4ec6-43f1-a6c1-d26511c96000)
Which means the config file is not even being applied. 

Autofix is set at false by default : 
![image](https://github.com/ai-cfia/github-workflows/assets/73133692/782b10bc-8ebf-47a0-952c-49c356c16a0e)

Reply:
yes, but again, even if it was doing that I think it would also require a step in the workflow to commit it back to the PR with the fixes. automatically creating commits without human intervention is bad imo.

Reply:
Yes im just explaining that the config file is not being applied and that the current behaviour is that the workflow only validates line lint but doesnt do any action. Want me to remove the config file since default behaviour seems to be aligning with our needs ?

Reply:
yes please